internal/storage/disk/disk.go

```diff
@@ -96,15 +96,15 @@ func (s *Storage) Open(dir string, options config.Badger) error {
 	}
 
 	// max size of lsm tree in bytes after which data is propagated to disk.
-	// The default is 64 MB.
+	// The default is 2 MB.
```

Comment:
How does it perform?

Reply:
Oh, in fact, this maybe a typo before. If you check the default of badger, it is 2MB. @kelindar

Reply:
I just update the comment.

Reply:
I won't introduce breaking changes or compatibility problems if possible